### PR TITLE
Add newlines to XPath sections

### PR DESCRIPTION
**Pull request by CanOfBees**
What Does this Do?
==================
This adds newlines between `` :code:`whatever` OR `` sections, to improve legibility in the HTML and PDF presentations.

How Should This Be Tested?
==========================
* Do all tests pass?
* How do the resulting documents look?

### CODE REVIEW

**Comment by markpbaggett** [comment]
Dead serious:  why double returns instead of just one? Am I being really dumb here?

File: docs/source/contents/4_draft_mapping.rst [hunk 1]
```diff
@@ -124,21 +124,53 @@ XPath
...
 :code:`identifier[@type="filename"]` OR
+
+
 :code:`identifier[@type="film number"]` OR
+
+
 :code:`identifier[@type="legacy"]` OR
+
+
 :code:`identifier[@type="local"]` OR
```
Comment (markpbaggett) [inline docs/source/contents/4_draft_mapping.rst]: Dead serious:  why double returns?

Comment (CanOfBees) [reply docs/source/contents/4_draft_mapping.rst]: PEBKAC. I... double-applied a regex? Honestly I'm not sure how I screwed this up, but thanks for asking: I found another screwup with your help.

**Review by markpbaggett — APPROVED** [review]
👍🏻

**Review by mlhale7 — APPROVED** [review]

**Pull request merged**